deploy/charts/openshift-routes/templates/deployment.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ include "openshift-routes.fullname" . }}
+  name: {{ include "openshift-routes.fullname" . }}-controller
```

Comment:
This change is necessary to circumvent the following issue:
```
$ helm install openshift-routes -n cert-manager oci://ghcr.io/cert-manager/charts/openshift-routes --version=0.5.0
$ make helm-chart VERSION=v0.6.0-alpha.1
$ helm upgrade openshift-routes -n cert-manager _bin/scratch/image/openshift-routes-0.6.0-alpha.1.tgz
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/tramlot/.kube/config
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/tramlot/.kube/config
Error: UPGRADE FAILED: cannot patch "openshift-routes" with kind Deployment: Deployment.apps "openshift-routes" is invalid: spec.selector: Invalid value: v1.LabelSelector{MatchLabels:map[string]string{"app.kubernetes.io/component":"controller", "app.kubernetes.io/instance":"openshift-routes", "app.kubernetes.io/name":"openshift-routes"}, MatchExpressions:[]v1.LabelSelectorRequirement(nil)}: field is immutable
```

Reply:
Indeed, seems like an annoying issue. Maybe we just leave the deployment name the same and ask users to use `helm upgrade --force`?

Reply:
I guess that is the easiest solution: not changing the label selector. Will try to achieve that.

Reply:
fixed in https://github.com/cert-manager/openshift-routes/pull/74/commits/80614e7bb7900c7937f981ec77536f1468f0c51e